doc/sphinx/proof-engine/ssreflect-proof-language.rst

```diff
@@ -3173,36 +3268,30 @@ the improved syntax for the control of rewriting.
 This may be a source of incompatibilities between the two rewrite
 tactics.
 
-In a rewrite tactic of the form:
-
-.. coqdoc::
-
-   rewrite occ_switch [term1]term2.
-
-``term1`` is the explicit rewrite redex and ``term2`` is the rewrite rule.
+In a rewrite tactic in the form ":n:`rewrite @occ_switch [@term__1] @term__2",
```

Comment:
I could not find where `@occ_switch` did go.

Reply:
`occ_switch` is no longer in the mlg grammar.  The `rewrite` has the syntax `rewrite {+ @r_prefix } {? @ssrclauses }`, which is in the chapter, but I couldn't guess how to update the syntax here.